shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt

```diff
@@ -40,40 +39,47 @@ interface ScreenshotTest {
         activity: Activity,
         heightInPx: Int? = null,
         widthInPx: Int? = null,
+        name: String? = null,
         backgroundColor: Int = android.R.color.white
     ) {
         val view = activity.findViewById<View>(android.R.id.content)
 
         if (heightInPx == null && widthInPx == null) {
             disableFlakyComponentsAndWaitForIdle(view)
-            takeActivitySnapshot(activity)
+            takeActivitySnapshot(activity, name)
         } else {
             runOnUi {
                 view.setBackgroundResource(backgroundColor)
             }
-            compareScreenshot(view = view!!, heightInPx = heightInPx, widthInPx = widthInPx)
+            compareScreenshot(view = view!!, heightInPx = heightInPx, widthInPx = widthInPx, name = name)
         }
     }
 
     fun compareScreenshot(
         fragment: Fragment,
         heightInPx: Int? = null,
-        widthInPx: Int? = null
-    ) = compareScreenshot(fragment.view!!, heightInPx)
+        widthInPx: Int? = null,
+        name: String? = null
+    ) = compareScreenshot(view = fragment.view!!, heightInPx = heightInPx, name = name)
```

Comment:
Unrelated: `widthInPx` isn't passed here. Is it intended?

Reply:
You are absolutely right @AlexKrupa this is a bug.